tests/unit/scp_test.py

```diff
@@ -57,3 +58,45 @@ def test_get(dst_path, src_path, ssh_scp, transmit_payload):
     """Check that SCP file download works."""
     ssh_scp.get(str(src_path), str(dst_path))
     assert dst_path.read_bytes() == transmit_payload
+
+
+@pytest.fixture
+def dst_path_exists(tmp_path):
+    """Return a local path that exists."""
+    path = tmp_path / 'non-existing.txt'
+    path.write_bytes('whatever'.encode())
```

Comment:
just use a byte string right away
```suggestion
    path.write_bytes(b'whatever')
```